fiduceo_uncertainties.f90

```diff
@@ -2027,11 +2025,18 @@ SUBROUTINE fill_netcdf(filename,AVHRR,FCDR)
     call check(nf90_inq_varid(ncid,"u_non_random_Ch5",ch5_us_varid))
     !print * ,"43"  
 
-    call check( nf90_put_var(ncid, ch1_us_varid, 0.03*AVHRR%new_array1, start = start_2))! count = count_pixel) )
+!MT: 07-11-2017: structured uncertainties on the reflectances are wrong
```

Comment:
This also looks like Marines old code

Reply:
Thank you. I have replaced the constant structured uncertainty values of 0.03 and 0.05 for the reflectance channels with the newly allocated variables:

-->     
call check( nf90_put_var(ncid, ch1_us_varid, ch1_non_random, start = start_2))
call check( nf90_put_var(ncid, ch2_us_varid, ch2_non_random, start = start_2))
call check( nf90_put_var(ncid, ch3a_us_varid, ch3a_non_random, start = start_2))